### PR TITLE
[All] Update to Compose 1.0.3

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: AccessibilityCodelab/build.gradle [hunk 1]
```diff
@@ -15,18 +15,18 @@
  */
 
 buildscript {
-    ext.kotlin_version = '1.5.10'
-    ext.compose_version = '1.0.0'
-    ext.coroutines_version = '1.4.2'
-    ext.accompanist_version = '0.15.0'
+    ext.kotlin_version = '1.5.30'
+    ext.compose_version = '1.0.3'
```
Comment (JolandaVerhoef) [inline AccessibilityCodelab/build.gradle:19]: Sorry for not updating this codelab before!

File: AdvancedStateAndSideEffectsCodelab/app/build.gradle [hunk 2]
```diff
@@ -113,18 +118,23 @@ dependencies {
     implementation Libs.AndroidX.Activity.activityCompose
     implementation Libs.AndroidX.appcompat
     implementation Libs.AndroidX.Compose.runtime
+    implementation Libs.AndroidX.Compose.runtimeLivedata
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply AdvancedStateAndSideEffectsCodelab/app/build.gradle:121]: The previous `build.gradle` configuration was having issues with SDK 31, so I just copy pasted the new one from Crane

File: AdvancedStateAndSideEffectsCodelab/app/build.gradle [hunk 3]
```diff
@@ -30,22 +30,27 @@ if (rootProject.file("local.properties").exists()) {
...
-        targetSdkVersion 30
+        targetSdkVersion 31
         versionCode 1
         versionName "1.0"
         vectorDrawables.useSupportLibrary = true
         testInstrumentationRunner "androidx.compose.samples.crane.CustomTestRunner"
 
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments["dagger.hilt.disableModulesHaveInstallInCheck"] = "true"
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply AdvancedStateAndSideEffectsCodelab/app/build.gradle]: Not needed, removed thanks!

File: AdvancedStateAndSideEffectsCodelab/app/build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply AdvancedStateAndSideEffectsCodelab/app/build.gradle]: Removed the unused ones, thanks

**Pull request merged**